src/praisonai/praisonai/cli/features/onboard.py

```
@@ -476,11 +635,13 @@ def _mask(value: str) -> str:
                     else:
                         console.print(
-                            "  [yellow]⚠  Warning:[/yellow] no allowlist set — anyone who finds your bot can use it."
+                            f"  [yellow]⚠  Warning:[/yellow] no allowlist set for {channel_key} — anyone who finds your bot can use it."
                         )
 
-                # Home-channel prompt intentionally removed — see PLATFORMS
-                # docstring above. No code path consumes home_channel today.
+        return env_to_save
+
+        # Step 2: Use the new token configuration method
+        env_to_save = self._configure_tokens(console, Prompt)
```

Ruff (0.15.14)

[error] Undefined name `Prompt`

(F821)